sys-auth/libfprint/libfprint-1.90.7.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson udev
+
+DESCRIPTION="library to add support for consumer fingerprint readers"
+HOMEPAGE="https://cgit.freedesktop.org/libfprint/libfprint/ https://github.com/freedesktop/libfprint https://gitlab.freedesktop.org/libfprint/libfprint"
+SRC_URI="https://github.com/freedesktop/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="2"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="examples +introspection"
+
+RDEPEND="dev-libs/glib:2
+	>=dev-libs/libgusb-0.3.1
+	dev-libs/nss
+	virtual/libusb:1=
+	x11-libs/gtk+:3
+	x11-libs/pixman
+	x11-libs/libX11
+	x11-libs/libXv
```

Comment:
Please sort this block.

Reply:
Sorted but I haven't pushed yet, waiting for your reply on other comments